mmv1/third_party/terraform/resources/resource_bigquery_table.go

```diff
@@ -605,6 +605,16 @@ func resourceBigQueryTable() *schema.Resource {
 							Optional:    true,
 							Description: `The maximum number of bad records that BigQuery can ignore when reading data.`,
 						},
+						// ConnectionId: [Optional] The connection specifying the credentials
+						// to be used to read external storage, such as Azure Blob,
+						// Cloud Storage, or S3. The connectionId can have the form
+						// "<project_id>.<location_id>.<connection_id>" or
+						// "projects/<project_id>/locations/<location_id>/connections/<connection_id>".
+						"connection_id": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: `The connection specifying the credentials to be used to read external storage, such as Azure Blob, Cloud Storage, or S3. The connectionId can have the form "<project_id>.<location_id>.<connection_id>" or "projects/<project_id>/locations/<location_id>/connections/<connection_id>".`,
```

Comment:
also nit: We've typically used `{{`/`}}` to show template strings in the provider, so I'd suggest that. As well, `project_id` and `location_id` generally have the `_id` dropped i.e.

`{{project}}.{{location}}.{{connection_id}}`

Reply:
done